docs/application/dotnet/guides/multimedia/metadata.md

```diff
@@ -252,8 +252,8 @@ The metadata is available with the properties and methods of the [Tizen.Multimed
 | Conductor            | Conductor of the content               | `Metadata.Conductor`                     |
 | Unsync lyrics        | Asynchronous lyrics of the content     | `Metadata.UnsyncLyrics`                  |
 | Sync lyrics          | Synchronous lyrics of the content      | `Metadata.SyncLyricsCount`<br>`MetadataExtractor.GetSyncLyrics()`<br>`SyncLyrics.Lyrics`<br>`SyncLyrics.Timestamp` |
-| Recorded date        | Description                            | Property or method                       |
-| Duration             | Duration of the content                | `Metadata.Duration`                      |
+| Recorded date        | Description                            | `Metadata.DateRecorded`                  |
```

Comment:
line 255: Fill in the right value. (ref. TD Recorded date, Recorded date of the content, Metadata.DateRecorded)
line 256: Fill in the right value. (ref. TD Orientation, Orientation of the content, Metadata.Rotation)